plugins/Redirector/class.redirector.plugin.php

```diff
@@ -385,7 +385,7 @@ public static function t5Filter(&$get) {
             if (val('_arg3', $get) == 'page') {
                 $result['_arg4'] = 'Page';
             }
-        } elseif (val('_arg1', $get) == 'bd-p') { // Board = Category
+        } elseif (val('_arg1', $get) == 'bd-p' || $get['_arg1'] == 'ct-p') { // bd-p ,ct-p ,tkb-p are Category, but pending for tkb-p
```

Comment:
What do you think of something like this instead.

Also, never put the comment at the end of a line. As per our standards, a line cannot exceed 80 characters. Always put the comment on the line above. It also improves readability.

```suggestion
        } elseif (isset($get['_arg1']) && in_array($get['_arg1'], ['bd-p', 'ct-p'])) {
```

Reply:
Actually, in here, I have a question for KB. Lithium kb type of DIscussion will have ```www.urlxxxx/tkb-p/xxxxx.com``. Should I add that in the in_array as well? Or we will handle Kb redirect differently?